jax/_src/pallas/triton/lowering.py

```diff
@@ -253,7 +253,6 @@ def lower_jaxpr_to_triton_module(
     in_shapes,
     grid_mapping: GridMapping,
     name: str,
-    cuda_options: Any,
 ) -> LoweringResult:
   # TODO(slebedev): Use cuda_options= during lowering.
```

Comment:
I would remove this comment as well at this point

Reply:
Ooops, good catch. This slipped through.